### ISSUE TITLE
Is the test label credible ？

### ISSUE THREAD

**Issue by GabrielXia** [fh295/semanticCNN#1]
Hello @fh295,

Thanks for the wonderful job you have done ! 

I would like to use the `ILSVRC2012_test_ground_truth.txt`, because I didn't find the labels in imagenet official site, I'm wondering if the test ground truth txt is credible. Thanks for you response !!

**Comment by RobinVogel** [comment]
I don't think it is, it is not available anywhere, and based on my tests, the performance of ResNets on these test labels is terrible. If the repository's owner fh295 passes by, can he delete these labels ?
If they are credible, does he has any idea as to why my tests failed ?

**Comment by dhanush-ai1990** [comment]
No these labels are not credible. I dont see any match with 2012 validation actual images from tar and the labels mentioned here
